### ISSUE TITLE
Reproducing results stated in the paper

### ISSUE THREAD

**Issue by aadith-warrier** [kakaxi314/BP-Net#26]
Hi Jie, 

Thank you for making your work open-source. I was trying to reproduce the results in the paper, using my own inference pipeline unfortunately the metrics are a bit off. Could you please confirm if my code is correct. 

```python
norm = transforms.Normalize(mean= [117, 97, 91], std=[70, 71, 74])
rgb, depth, raw_rgb, dataset, tom_mask = batch
rgb = norm(raw_rgb) # raw_rgb is an image tensor with range from [0,255]
rgb = rgb.to(device)
raw_rgb = raw_rgb.to(device)
depth = depth.to(device)

sample_depth = random_sample_mask(depth, 500, tom_mask.unsqueeze(0))
sparse_depth = torch.where(sample_depth==1, depth, torch.tensor(0, dtype=torch.float32).to(device))
sparse_depth = sparse_depth.to(device)
pred_depth = net(rgb, sparse_depth*10, Kcam.unsqueeze(0).to(device))
pred_depth = pred_depth[-1]/10
```

I scale the inputs by 10 before feeding to the network as this is done [here](https://github.com/kakaxi314/BP-Net/blob/bb1886bdf454d3229a8d3d2aedc75476e9621084/datasets/nyu.py#L176).  

Currently on the validation set of NYU Depth V2, I have the following results. 
RMSE: 0.281041
REL: 0.028775
